### ISSUE TITLE
"How to plan a lesson..." section is visually presented as a list, but isn't in HTML

### ISSUE THREAD

**Issue by jack-skerrett-oak** [oaknational/Oak-Web-Application#1404]
On lesson planning, the "1. Find the right lesson, 2. Tailor...." looks as though it should be a list, but the HTML has it as a whole load of divs

**Comment by jack-skerrett-oak** [comment]
https://www.notion.so/oaknationalacademy/The-content-within-the-How-to-plan-a-lesson-using-our-resources-and-adapt-them-for-your-classroom--c8ae3498798a490fa8201cd4b096501b?pvs=4
